robyn/router.py

```diff
@@ -79,25 +89,41 @@ def add_route(
         handler: Callable,
         is_const: bool,
         exception_handler: Optional[Callable],
+        default_response_headers: List[Header],
     ) -> Union[Callable, CoroutineType]:
+        headers_list = (h.as_list() for h in default_response_headers)
+        response_headers = {d[0]: d[1] for d in headers_list}
```

Comment:
```suggestion
        response_headers = {h.key: h.val for h in default_response_headers}
```

Hey @IdoKendo 👋 

Can we do this instead?